javascript/javascript2/week1/homework/cardio.js

```diff
@@ -0,0 +1,43 @@
+//FINDS THE SHORTEST WORD
+function findShortestWord(array) {
+  shortestWord = array.reduce(function (prev, next) {
```

Comment:
Super clever use of reduce! We haven't even learned array methods yet, so this is super impressive! 
 
If you want to make the function more concise, you could equivalently write: 
```javascript 
function(prev, next) {
    if (prev.length <= next.length) {
      return prev;
    } else { return next }
}
```

And if you really wanted to make it as short as possible, you could use a ternary expression like so:  
```javascript 
function(prev, next) { 
     return prev.length <= next.length ? prev : next; 
 } 
```

Reply:
I've just watched videos for the next class from preparations materials and thought that I could have done this away shorter😅